selftests/functional/test_basic.py

```diff
@@ -1,3 +1,4 @@
+# This Python file uses the following encoding: utf-8
```

Comment:
Oops, I missed that second example on the same PEP.  They're all OK, and the second example is more informative.